cloudbridge/cloud/providers/gce/provider.py

```diff
@@ -82,6 +82,8 @@ def __init__(self, connection):
 
         # We will not mutate self._desc; it's OK to use items() in Python 2.x.
         for resource, resource_desc in desc['resources'].items():
+            if 'methods' not in resource_desc:
```

Comment:
Don't know if this reads better but this could also be:

```
if not resource_desc.get('methods', {}).get('get'):
  continue
```

or

```
methods = resource_desc.get('methods', {})
if not methods.get('get'):
  continue
```

Also - it isn't clear why this block of code is here. What is the check for the "get" method for here? It is not referenced in the code here.